modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ViewObjectDefinitions/ViewObjectDefinitions.tsx

```diff
@@ -364,7 +371,14 @@ export default function ViewObjectDefinitions({
 								}
 								viewMode="no-header-border"
 							>
-								<FrontendDataSet {...dataSetProps} />
+								{reloadFDS ? (
+									<ClayLoadingIndicator
+										displayType="secondary"
+										size="sm"
+									/>
+								) : (
+									<FrontendDataSet {...dataSetProps} />
+								)}
```

Comment:
@Artartur, great! But we also have to use this same code when the feature flag is disabled in line `387` since we are not using the `reload` prop from the modal anymore.

Reply:
Done!